www-src/app/replicator/fromDevice/change_synchronizer.coffee

```diff
@@ -0,0 +1,12 @@
+EventSynchronizer = require "./event_synchronizer"
+log = require('../../lib/persistent_log')
+    prefix: "ChangeSynchronizer"
+    date: true
+
+module.exports = class ChangeSynchronizer
```

Comment:
We may call it ChangesImporter , making the direction ( fromDevice ) more explicit. (to discuss)